### PR TITLE
Add solution for the hangouts chat codelab

### PR DESCRIPTION
**Pull request by erickoledadevrel**

### CODE REVIEW

File: call-in-sick/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+1.  Open [Hangouts Chat][hangouts_chat].
+1.  Click **Find people, rooms, bots > Message a bot**.
+1.  From the list, select the **Attendance Bot** that you created.
+1.  Send the message "I'm sick" to the bot.
+1.  Click the **CONFIGURE** button and authorize access to your account.
+1.  In the resulting card, click the **SET VACATION IN GMAIL** and
+    **BLOCK OUT DAY IN CALENDAR** buttons.
+
+[code]: https://script.google.com/d/1pbuGhMkTyqfeR30QfbMzsA21qM2p0HmnStGpZUKj-QLvH0BL73UbTlSq/edit
```
Comment (asrivas) [inline call-in-sick/README.md]: Should this be /copy and we can have the users make a copy of the script in their hub?

File: call-in-sick/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+First, create the script:
+
+1.  [Click here][code] to open the bot code in the Apps Script code editor.
+1.  Click the menu item **File > Make a copy** to get your own copy. You can
+    later access this copy in the [G Suite Developer Hub][hub].
+
+Then create a Google Cloud project to use with it:
+
+1.  [Create a new project][new_project] in the Google Cloud Developer Console.
+    Name it "Call in Sick", select a Billing Account if prompted, and
```
Comment (asrivas) [inline call-in-sick/README.md]: Does Billing Account need to be bolded? 
https://developers.google.com/style/ui-elements

Comment (asrivas) [inline call-in-sick/README.md]: Should we link a guide on how to set up a billing account? Looking at some of the links in GCP codelabs like:
https://console.developers.google.com/billing/freetrial?hl=en
https://cloud.google.com/free/docs/gcp-free-tier

File: call-in-sick/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+1.  In the field **Application name** enter "Attendance Bot" and click the
+    **Save** button at the bottom.
+1.  Open the [**Gmail API**][library_gmail] page in the API library and click
+    the **ENABLE** button.
+1.  Open the [**Project settings**][project_settings] page for the project.
+1.  Copy the value listed under **Project number**.
+
+Next, setup your script to use the new project and deploy it:
+
+1.  Back in the App Script Code Editor, click the menu item
```
Comment (asrivas) [inline call-in-sick/README.md]: nit: Apps

File: call-in-sick/README.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    **Resources > Cloud Platform project**.
+1.  Enter the project number into the text box and click **Set Project**. When
+    prompted, click Confirm.
```
Comment (asrivas) [inline call-in-sick/README.md]: bold Confirm

File: call-in-sick/README.md (hunk 2 above)
Comment (erickoledadevrel) [reply call-in-sick/README.md]: Done

Comment (erickoledadevrel) [reply call-in-sick/README.md]: Done

File: call-in-sick/README.md (hunk 3 above)
Comment (erickoledadevrel) [reply call-in-sick/README.md]: Done

File: call-in-sick/README.md (hunk 1 above)
Comment (erickoledadevrel) [reply call-in-sick/README.md]: As mentioned offline, this doesn't work for scripts.

**Pull request merged**